draft-ietf-quic-recovery.md

```diff
@@ -418,6 +418,12 @@ used to send a probe into the network prior to establishing any packet loss,
 prior unacknowledged packets SHOULD NOT be marked as lost when a TLP timer
 expires.
 
+If no new data or unacknowledged data is available to send, a retransmittable
+frame SHOULD be sent.  Sending a retransmittable frame ensures that any in
+flight packets are acknowledged or declared lost in a timely manner,
```

Comment:
Well, if you TLP, then you haven't received an ACK that contained any information about the in flight packets, because otherwise you'd fast or early retransmit them.  So you have some number of in flight packets, and you don't know what their status is.  It's possible the ACK was just lost, etc.  So in that sense, it accelerates receiving an acknowledgement for them if they were in fact not lost.

Reply:
Ah yes, ack losses. You're right